python/lsst/sims/GalSimInterface/galSimInterpreter.py

```diff
@@ -1170,6 +1170,16 @@ def getGoodPhotImageSize(obj, keep_sb_level, pixel_scale=0.2):
     N = obj.getGoodImageSize(pixel_scale)
     #print('N = ',N)
 
+    try:
```

Comment:
I'm having a hard time understanding why this code only works for random walks.  I think it might be good if you could add a bit more explanation in the comments.

Reply:
Is this sufficient?  c4f1d73

Reply:
Yes, thanks. That's helpful.   I looked around and  it seems currently only correlated noise objects also have a _profile.  

Is this something that might show up in other classes of objects in the future?

Reply:
No.  I don't see any reason for any other gsobject class to start using this attribute.